py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -129,7 +129,8 @@ def _parse_config(self, config):
 
         self.num_processors = config.getint("num processors")
 
-        self.var_lss_binning = config.get("var lss binning", 'log')
+        self.raw_statistics_binning = config.get("raw statistics binning", 'log')
```

Comment:
Also, why do you need to specify the pixel? Isn't this something that you would get from the raw_statistics_file?

Reply:
There are three raw_statistics_files that can be read right now. One with log pixelization and two with two linear schemes. I think Alma implemented it in this way because the pixelization you use in your delta_extraction run could not be in any of the raw_statistics_files. In that case you may want to select the one that agrees the most.

Reply:
Regarding the log pixelization, since that is the default pixelization in pica (until now at least), it is good to preserve it as default also here(?)

Reply:
Please keep the log pixelization, I'm using it for other projects with Andrea, we'll eventually move to the linear pixelization but not yet.

Reply:
Ok, let's leave it for now.